### PR TITLE
ENGDOCS-2207

### PR DESCRIPTION
**Pull request by aevesdocker**
## Description

Adds multi-org support, RAM precedence and a few examples where appropriate

Previews: 
https://deploy-preview-20885--docsdocker.netlify.app/security/for-admins/enforce-sign-in/methods/
https://deploy-preview-20885--docsdocker.netlify.app/security/for-admins/hardened-desktop/registry-access-management/

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

**Review by KatTomrushka — APPROVED** [review]
Looks good - thanks! 

One potential improvement could be to state the exact registry type is REG_MULTI_SZ in point 2 of Registry key method (Windows only) section here https://deploy-preview-20885--docsdocker.netlify.app/security/for-admins/enforce-sign-in/methods/

File: layouts/shortcodes/admin-registry-access.html [hunk 1]
```diff
@@ -15,13 +15,17 @@
...
+Once you add a registry, it can take up to 24 hours for the changes to be enforced on your developers’ machines. 
 
-   > [!TIP]
-   >
-   > Since RAM sets policies about where content can be fetched from, the [ADD](/reference/dockerfile/#add) instruction of the Dockerfile, when the parameter of the ADD instruction is a URL, is also subject to registry restrictions. It's recommended that you add the domains of URL parameters to the list of allowed registry addresses under the Registry Access Management settings of your organization.
+If you want to apply the changes sooner, you must force a Docker signout on your developers’ machine and have the developers re-authenticate for Docker Desktop. See the Caveats section below to learn more about limitations when using this feature.
```
Comment (dvdksn) [inline layouts/shortcodes/admin-registry-access.html:23]: I suggest we link to the caveats section

```suggestion
If you want to apply the changes sooner, you must force a Docker signout on your developers’ machine and have the developers re-authenticate for Docker Desktop. See the [Caveats](#caveats) section below to learn more about limitations when using this feature.
```

File: layouts/shortcodes/admin-registry-access.html [hunk 2]
```diff
@@ -15,13 +15,17 @@
...
+
+> [!IMPORTANT]
+>
+> Starting with Docker Desktop version 4.35, you can enforce sign-in for multiple organizations. If a developer belongs to multiple organizations with different RAM policies, only the RAM policy for the first organization listed in the `registry.json` file, `.plist` file, or registry key is enforced.
+
+> [!TIP]
+>
+> Since RAM sets policies about where content can be fetched from, the [ADD](/reference/dockerfile/#add) instruction of the Dockerfile, when the parameter of the ADD instruction is a URL, is also subject to registry restrictions. It's recommended that you add the domains of URL parameters to the list of allowed registry addresses under the Registry Access Management settings of your organization.
```
Comment (dvdksn) [inline layouts/shortcodes/admin-registry-access.html:31]: - Why is this a tip?
- I think this needs more explanation. What does `ADD <url>` have to do with registries? You are never accessing a registry with that instruction: it's either the build context, a Git repository, or a regular HTTP(S) URL.
  - Does it also affect `RUN` instructions, e.g. `RUN wget https://example.com/foo.tgz`?
- What about: `FROM <image>`, `COPY --from=<image>`, `RUN --mount=from=<image>`
- Does this also apply for DBC builders and remote builders? The caveats section mentions `kubernetes` and `docker-container`, but not `remote` and `cloud`.

**Review by dvdksn — APPROVED** [review]

**Comment by stanislavHamara** [comment]
I don't know what the docs process is, is anything stopping this from being merged?

**Comment by KatTomrushka** [comment]
@stanislavHamara docs are published around the time the release happens. In this case, the feature was planned for DD 4.35 but this release will be delayed or entirely skipped to 4.36 (end of Oct) due to stability issues.

**Pull request closed without merging**